site/content/docs/browser-checks/_index.md

```diff
@@ -164,79 +151,70 @@ However, many times you want to assert specific values on a page.
-await page.goto('https://checklyhq.com/')
+test('CTA button has "Start for free" text', async ({ page }) => {
+  await page.goto('https://checklyhq.com/')
 
-// get the text of the button
-const buttonText = await page.$eval('a.btn-lg', el => el.innerText)
+  // CTA button locator
+  const button = page.locator('#nav-signup-button')
 
-// assert using Jest's expect function
-expect(buttonText).toEqual('Start for free')
-
-await browser.close()
+  // Assert that the button has the correct text
+  await expect(button).toHaveText('Start for free')
+})
  ```
 {{< /tab >}}
 {{< /tabs >}}
 
-Note the following:
-
-- We use the `page.$eval()` method to grab the button element and get its innerText property.
-- We use a basic Jest `expect().toEqual()` statement to verify the text is correct.
+Note that we are using Playwright Tests's built-in expect, which is enriched with a convenient [LocatorAssertions](https://playwright.dev/docs/api/class-locatorassertions) class. Methods of this class can be used to make assertions about `Locators` state. Here we use `toHaveText()` to check if the target element has `Start for free` text.
 
 When an assertion fails, your check fails. Your check's result will show the log output for the error. Any configured
 alerting channels will be triggered, notifying your team that something is up.
 
-![failed api monitoring assertion](/docs/images/browser-checks/failed_assertion.png)
+![failed PWT assertion](/docs/images/browser-checks/getting-started_pwt.gif)
 
+## What about Puppeteer?
+While Playwright and Puppeteer share many similarities, they have evolved at different speeds over time. Playwright's rapid release cycle and new features such as [auto-waiting](https://playwright.dev/docs/actionability) and [the built-in inspector](https://playwright.dev/docs/debug#playwright-inspector) made it gain momentum in the developer community. Playwright and Playwright Test Runner have become superior solutions and we have stopped support for Puppeteer in newer [runtimes](/docs/runtimes/). The latest runtime that supports Puppeteer is [2022.02](/docs/runtimes/specs/).
 
-## Next Steps
+We recommend using Playwright Test if you are just starting out or [migrating from Puppeteer to Playwright using `puppeteer-to-playwright`](https://github.com/checkly/puppeteer-to-playwright).
 
+## Next Steps
+- Learn more about [built-in functionalities of Playwright Test](/docs/browser-checks/playwright-test/).
 - Learn how to deal with [login scenarios and private data](/docs/browser-checks/login-and-secrets/).
 - Use [Playwright Codegen](https://playwright.dev/docs/codegen) to record scripts without coding.
 - Learn more about [taking screenshots](/docs/browser-checks/screenshots/).
 - Learn more about [creating reusable code snippets](/docs/browser-checks/partials-code-snippets/).
 - Run checks behind an [HTTP proxy](/docs/private-locations/proxy).
 
-## More Playwright and Puppeteer resources
+## More Playwright resources
```

Comment:
@miliberlin should we also link our Youtube channel? It is very Browser check / PWT focused right now so I think it would fit well.